source/_integrations/plex.markdown

```diff
@@ -28,6 +28,10 @@ If a Plex server has been claimed by a Plex account via the [claim interface](ht
 
 The Plex integration is set up via **Configuration** -> **Integrations**. During the configuration, you will be redirected to the [Plex](https://plex.tv) website to sign in with your Plex account. Once access is granted, Home Assistant will connect to the Plex server linked to the provided account. If multiple Plex servers are available on the account, you will be prompted to complete the configuration by selecting the desired server on the **Integrations** page. Upon completeion, Home Assistant will be displayed as an authorized device on the [Plex Web](https://app.plex.tv/web/app) interface under **Settings** -> **Authorized Devices**.
 
+During setup the integration will check all possible ways to connect to your Plex server(s) - i.e. the local network IP address or the public IP address of your internet connection, HTTP or HTTPS, directly or via the domain plex.direct, and chose local over public and secure over insecure. You can check your HA log to see which connection was chosen. If your internet router uses DNS rebind protection the direct connect to the local address via plex.direct will fail (see [Plex documentation](https://support.plex.tv/articles/206225077-how-to-use-secure-server-connections/) - scroll way down to DNS Rebinding). To avoid this configure your router to whiteliste the domain plex.direct if possible.
```

Comment:
```suggestion
During setup, the integration will check all possible ways to connect to your Plex server(s) - i.e., the local network IP address or the public IP address of your internet connection, HTTP or HTTPS, directly or via the domain `plex.direct`, and chose local over public and secure over insecure. You can check your HA log to see which connection was chosen. If your internet router uses DNS rebind protection, the direct connection to the local address via `plex.direct` will fail (see [Plex documentation](https://support.plex.tv/articles/206225077-how-to-use-secure-server-connections/) - scroll way down to DNS Rebinding). To avoid this, configure your router to allow the domain `plex.direct`, if possible.
```